src/components/index.ts

```diff
@@ -46,7 +46,7 @@ export {
   HSBLAColor,
   rgbToHex,
   rgbToHsb,
-  rgbToHsl,
+  rgbToHsla,
```

Comment:
I really hope nobody has been using this publicly, but this is a breaking change